pywwt/layers.py

```diff
@@ -483,13 +495,7 @@ def _table_b64(self):
         # TODO: We need to make sure that the table has ra/dec columns since
         # WWT absolutely needs that upon creation.
 
-        s = StringIO()
-        self.table.write(s, format='ascii.basic', delimiter=',', comment=False)
-        s.seek(0)
-
-        # Enforce Windows line endings
-        # TODO: check if this needs to be different on Windows
-        csv = s.read().replace('\n', '\r\n')
+        csv = csv_table_win_newline(self.table)
```

Comment:
Does the behavior need to be different on Windows vs other platforms? I'm not sure why the code was the way it was before, but it seems to me that we only need to fix newlines when on Windows? (in which case we could use the ``platform`` module to check).

Reply:
WWT requires CSV data to use Windows line endings, but Astropy writes using the native line endings. The code that was there was designed to convert Mac line endings to Windows, but it messed up the line endings on Windows.